controllers/oauth.js

```diff
@@ -42,6 +42,10 @@ function init(req, res, next, client = FxAOAuthClient) {
   req.session.state = state;
   const url = new URL(client.code.getUri({state}));
   url.searchParams.append("access_type", "offline");
+  url.searchParams.append("action", "signin");
```

Comment:
I think we decided this should be `action=email` to trigger the "email firs" flow on accounts.firefox.com

Reply:
Ok cool-  Closing this to open a PR for the fxa-updates branch and will make the updates there.